src/appleseed.studio/mainwindow/project/projectbuilder.h

```diff
@@ -165,15 +165,11 @@ Entity* ProjectBuilder::edit_entity(
     return new_entity_ptr;
 }
 
-template <>
-inline renderer::EnvironmentEDF* ProjectBuilder::edit_entity(
-    renderer::EnvironmentEDF*           old_entity,
-    renderer::Scene&                    parent,
-    const foundation::Dictionary&       values) const
+template <typename OldEntity, typename NewEntity>
+inline void copy_transform_sequence(
```

Comment:
Is it okay to define `copy_transform_sequence()` between `edit_entity()` definitions?